### PR TITLE
WIP - Handle only postable resource

### PR DESCRIPTION
**Pull request by jfthuillier**
Fix issue #53

### CODE REVIEW

File: src/hydra/parseHydraDocumentation.js [hunk 1]
```diff
@@ -7,6 +7,16 @@ import Operation from "../Operation";
...
 import getParameters from "./getParameters";
 
+/**
+ * Extracts the url of a resource.
+ *
+ * @param {string} propertyId
+ * @return {string}
+ */
+function guessUrlFromPropertyId(propertyId) {
+  return `${propertyId.replace("docs.jsonld#Entrypoint/", "")}s`;
```
Comment (dunglas) [inline src/hydra/parseHydraDocumentation.js]: This hardcoded replacement creates a hard coupling with API Platform Core while we want this lib to work with any Hydra implementation.

The base URL can be extracted from `hydra:entrypoint` (and is probably available somewhere as a var).

The resource name could be extracted using the `URL` API (hash part) and a regex, even.

Some plural don't end by a `s`, this shouldn't be hardcoded as well.

File: src/hydra/parseHydraDocumentation.test.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ const entrypoint = `{
     "deprecatedResource": {
       "@id": "Entrypoint/deprecatedResource",
       "@type": "@id"
+    },
+    "onlyPostResource": {
+      "@id": "Entrypoint/onlyPostResource",
```
Comment (dunglas) [inline src/hydra/parseHydraDocumentation.test.js]: `post only` would sounds more English

**Comment by dunglas** [comment]
I think that it should be fixed in API Platform Core first.

1. In the API entrypoint, (`/index.jsonld`), every properties should be listed, even if now GET operations are associated with them (AFAIU, it's currently not the case, and it's wrong).
2. In the API documentation (`/docs.jsonld`), this new property of the entrypoint must be marked with `readable: false` and `writeable: true`, as it's indeed a write-only property of the API entrypoint.
3. We have to check that this is supported in this lib

Doing this would allow to remove most hardcoded code, and will be cleaner. WDTY?

**Comment by jfthuillier** [comment]
I agree, i had the same feeling that this solution was "hacky". I'll consider your comment and investigate  into the core.

**Comment by Gradius30** [comment]
Hello guys. Is there any progress to this PR? 
We are currently using ApiPlatform and the fetch always fails on the mentioned line.

**Pull request closed without merging**